release_tester/arangodb/installers/nsis.py

```diff
@@ -212,3 +214,46 @@ def cleanup_system(self):
             shutil.rmtree(self.cfg.appdir)
         if self.cfg.cfgdir.exists():
             shutil.rmtree(self.cfg.cfgdir)
+        try:
+            k = winreg.OpenKeyEx(winreg.HKEY_LOCAL_MACHINE,
+                                 "SYSTEM\\CurrentControlSet\\Services\\ArangoDB")
+            winreg.CloseKey(k)
+
+            print("force deleting the arangodb service")
+            with winreg.OpenKeyEx(winreg.HKEY_LOCAL_MACHINE,
+                                  "SYSTEM\\CurrentControlSet\\Services",
+                                  access=winreg.KEY_WRITE) as k:
+                winreg.DeleteKey(k, "ArangoDB")
+                psutil.Popen(["sc", "delete", "arangodb"]).wait()
```

Comment:
Shouldn't this be enough? Is it really necessary to delete the registry key by hand?

Reply:
hm, probably these were the first sights of service handle leaks, we probably can remove that again.